homeassistant/components/weheat/strings.json

```diff
@@ -25,7 +25,12 @@
       "user_rejected_authorize": "[%key:common::config_flow::abort::oauth2_user_rejected_authorize%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
       "no_devices_found": "Could not find any heat pumps on this account",
-      "wrong_account": "You can only reauthenticate this account with the same user."
+      "wrong_account": "You can only reauthenticate this account with the same user.",
+      "get_user_failed": "Could not retrieve user information from Weheat due to a HTTP error. Please try again later.",
+      "get_heat_pumps_failed": "Could not retrieve heat pumps due to a HTTP error. Please try again later.",
```

Comment:
Yea I think we should not bother the user with HTTP errors, because they can't do anything about that